### PR TITLE
Membership cancellation landing v2 cc

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
Making changes as per the design copy:
- Removed chat to us button
- Updated wording 

## Images
image

image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx [hunk 1]
```diff
@@ -89,42 +89,25 @@ export const MembershipCancellationLanding = () => {
 			<section css={sectionSpacing}>
 				<Stack space={3}>
 					<Heading>
-						We're sorry to hear you're thinking of cancelling.
+						We're sorry to hear you're thinking of leaving
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx:92]: all small nitpicks: I think the design has no top border on the heading?

File: client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx [hunk 2]
```diff
@@ -89,42 +89,25 @@ export const MembershipCancellationLanding = () => {
...
-					</h2>
+					<p>
+						To cancel today, please choose from the following
+						options.
+					</p>
 				</Stack>
 			</section>
 			<section css={sectionSpacing}>
 				<Stack space={3}>
 					<Heading sansSerif>Call us</Heading>
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx:102]: Call us "to cancel"

File: client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx [hunk 3]
```diff
@@ -89,42 +89,25 @@ export const MembershipCancellationLanding = () => {
...
 				</Stack>
 			</section>
 			<section css={sectionSpacing}>
 				<Stack space={3}>
 					<Heading sansSerif>Cancel online</Heading>
-					You can cancel online Lorem Lipsum.
+					Continue without speaking to our customer service team.
 					<div css={buttonLayoutCss}>
 						<Button
 							icon={<SvgArrowRightStraight />}
```
Comment (andrewHEguardian) [inline client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx:113]: I think the arrow is no longer in the designs

**Comment by charleycampbell** [comment]
[Membership Saves - Copy changes V2 - Landing page](https://trello.com/c/rxyuZKX4/4121-membership-saves-copy-changes-v2-landing-page)

**Review by andrewHEguardian — APPROVED** [review]
Looks good 🌟

**Pull request merged**